### ISSUE TITLE
Add accessibility section

### ISSUE THREAD

**Issue by LJWatson** [w3c/IndexedDB#327]
In the [WebApps charter](https://www.w3.org/2019/05/webapps-charter.html#scope) it says:
>Where there are implications for implementors, developers, or users, in the areas of accessibility, internationalization, privacy, and security, each specification must have a section that describes relevant benefits, limitations, and best practice solutions for that particular area.>

To this end each spec needs to have an accessibility section. For the avoidance of doubt in the reader, the section should be included even if it indicates there are no known accessibility considerations.

Relates to w3c/webappswg#15

**Comment by inexorabletash** [comment]
It should also have an Internationalization section, to round out the horizontals.

Are there any good templates to follow, or existing specs that speak to accessibility concerns well, especially for similar pure-API standards with no expected UI?

**Comment by LJWatson** [comment]
We're blazing a bit of a trail on this. For an API like this where there is definitely no impact on the UI/accessibility, I'd suggest an "Accessibility" heading followed  by a very short and simple section that says something like the following:
>This specification has no impact on the user interface and therefore there are no accessibility considerations for implementors, developers or users.>

**Comment by akuckartz** [comment]
> For an API like this where there is definitely no impact on the UI/accessibility ...

Are there any checklists regarding potential accessibility impacts of APIs?

**Comment by LJWatson** [comment]
There is the [Framework for Accessible Specification of Technologies (FAST)](https://w3c.github.io/apa/fast/) with an associated [FAST checklist](https://w3c.github.io/apa/fast/checklist.html).

Worth noting that in 2016 the APA WG (that does horizontal review for accessibility) decided that this spec [did not need accessibility review](https://www.w3.org/WAI/APA/wiki/Indexed_Database_API_2.0), presumably (if my memory serves) because it has no impact on the UI.

**Comment by inexorabletash** [comment]
Awesome, thanks for the links. The checklist will provide a good format, even if mostly enumerating that this API does not have most of the relevant features.

Note that user agents are likely to have some user interface elements associated with storage APIs even when the storage APIs do not define them - such as developer tools, or for end users to see which sites are storing data. I called that out in a draft PR.

**Comment by inexorabletash** [comment]
Updated draft, direct link to the new accessibility section is https://pr-preview.s3.amazonaws.com/w3c/IndexedDB/pull/328.html#accessibility

**Comment by inexorabletash** [comment]
Anyone want to review the PR?

**Comment by pwnall** [comment]
@inexorabletash Done.

**Comment by LJWatson** [comment]
The accessibility section is terrific! Thanks @inexorabletash
